sources/packages/forms/src/form-definitions/institutionwelcomepage.json

```diff
@@ -0,0 +1,3175 @@
+{
+    "forms": [
+        {
+            "title": "Institution Welcome Page",
+            "display": "form",
+            "type": "form",
+            "name": "institutionwelcomepage",
+            "path": "institutionwelcomepage",
+            "tags": [
+                "common"
+            ],
+            "components": [
+                {
+                    "collapsible": false,
+                    "hideLabel": true,
+                    "key": "panel",
+                    "type": "panel",
+                    "label": "Panel",
+                    "input": false,
+                    "tableView": false,
+                    "components": [
+                        {
+                            "label": "HTML",
+                            "tag": "div",
+                            "attrs": [
+                                {
+                                    "attr": "",
+                                    "value": ""
+                                }
+                            ],
+                            "content": "Welcome to your institution account!",
+                            "refreshOnChange": false,
+                            "customClass": "category-header-large primary-color mt-5 mb-5",
```

Comment:
Maybe you the html is changed to a "<p>" there is no need for the "mt-5 mb-5".
I believe that other forms are using more <p> in general.